### PR TITLE
ADR-1321: Spoilt journey amendment

### PR DESCRIPTION
**Pull request by sini-george**

### CODE REVIEW

File: app/controllers/adjustment/AdjustmentTypeController.scala [hunk 1]
```diff
@@ -100,4 +106,23 @@ class AdjustmentTypeController @Inject() (
         )
     }
 
+  private def checkIfOneRegimeAndUpdateUserAnswer(
```
Comment (davidfes) [inline app/controllers/adjustment/AdjustmentTypeController.scala:109]: Name doesn't help understand what it does. Type is UserAnswers (plural) in name/vars.

e.g. checkIfOneRegimeAnd....InUserAnswers ?

File: app/controllers/adjustment/SpoiltVolumeWithDutyController.scala [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                } yield Redirect(navigator.nextPage(SpoiltVolumeWithDutyPage, mode, updatedAnswers))
+              }
+            )
+        case _            =>
+          logger.warn("Couldn't fetch regime value in AdjustmentEntry from user answers")
+          Future.successful(Redirect(controllers.routes.JourneyRecoveryController.onPageLoad()))
+      }
+  }
+
+  private def negativeDuty(value: BigDecimal): BigDecimal =
```
Comment (davidfes) [inline app/controllers/adjustment/SpoiltVolumeWithDutyController.scala:139]: These don't seem used

File: app/viewmodels/govuk/LabelFluency.scala [hunk 3]
```diff
@@ -28,6 +28,12 @@ trait LabelFluency {
 
     def apply(content: Content): Label =
       Label(content = content)
+
+    def apply(visibleText: String, hiddenText: String): Label = {
+      val content = HtmlContent(s"$visibleText <span class='govuk-visually-hidden'>$hiddenText</span>")
```
Comment (davidfes) [inline app/viewmodels/govuk/LabelFluency.scala:33]: govuk-visually-hidden -> Constants ?

File: app/controllers/adjustment/SpoiltVolumeWithDutyController.scala (hunk 2 above)
Comment (sini-george) [reply app/controllers/adjustment/SpoiltVolumeWithDutyController.scala:139]: They are used on line 117

**Review by davidfes — APPROVED** [review]

**Review by davidfes — DISMISSED** [review]

**Review by davidfes — APPROVED** [review]

**Review by msatta — DISMISSED** [review]

**Review by msatta — APPROVED** [review]

**Pull request merged**